src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1838,8 +1856,10 @@ private void getCallInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_RESO
             pResult.kind = CORINFO_CALL_KIND.CORINFO_CALL;
             pResult._nullInstanceCheck = (uint)(((flags & CORINFO_CALLINFO_FLAGS.CORINFO_CALLINFO_CALLVIRT) != 0) ? 1 : 0);
 
-            // TODO: Generics
-            // pResult.contextHandle;
+            // TODO: Support Generics
+            //
+            // For now, set the contextHandle to zero to indicate Generics context is not supported.
+            pResult.contextHandle = (CORINFO_CONTEXT_STRUCT *)0;
```

Comment:
The contextHandle is used for more than just shared generics. We should just set it to the right (non shared generics) value here - and not have the null checks in other places.